account_invoice_line_report/report/account_invoice_report_view.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+     Copyright 2018 Komit <http://komit-consulting.com>
```

Comment:
You are not really modifying this file (only removing a commented line), so the copyright addition doesn't apply

Reply:
I changed those.